src/views/AnalysisModule/Analysis/index.tsx

```diff
@@ -16,16 +32,180 @@ interface ComponentProps {
+                <div className={styles.subHeading}>
+                    <span className={styles.boldText}>
+                        Creation Date
+                    </span>
+                    &nbsp;
+                    June 5, 2021
+                </div>
+            )}
+        >
+            <div className={styles.pillarBody}>
+                <div className={styles.left}>
+                    <div className={styles.item}>
+                        Analyst &nbsp;
+                        <span className={styles.boldText}>
+                            {assigneeName}
+                        </span>
+                    </div>
+                </div>
+            </div>
+        </Container>
+    );
+};
+
+const keySelector = (item: AnalysisPillars) => (item.id);
+
 function Analysis(props: ComponentProps) {
     const {
         title,
         className,
         startDate,
         endDate,
+        activeProject,
+        analysisId,
+        setAnalysisDeleteId,
     } = props;
 
+    const [analysisPillar, setAnalysisPillar] = useState<AnalysisPillars[]>([]);
+    const [pillarCount, setPillarCount] = useState<number>();
+    const [activePage, setActivePage] = useState<number>(1);
+    const [isExpanded, setIsExpanded] = useState<boolean>(false);
+    const [pillarAnalysisToDelete, setPillarAnalysisToDelete] = useState<number | undefined>();
+    const [
+        pillarPending,
+        ,
+        ,
+        pillarGetTrigger,
+    ] = useRequest<MultiResponse<AnalysisPillars>>(
+        {
+            url: `server://projects/${activeProject}/analysis/${analysisId}/pillars/`,
+            method: 'GET',
+            query: {
+                offset: (activePage - 1) * MAX_ITEMS_PER_PAGE,
+                limit: MAX_ITEMS_PER_PAGE,
+            },
+            autoTrigger: true,
+            onSuccess: (response) => {
+                setAnalysisPillar(response.results);
+                setPillarCount(response.count);
+            },
+        },
+    );
+
+    const [
+        ,
+        ,
+        ,
+        deletePillarTrigger,
+    ] = useRequest(
+        {
+            url: `server://projects/${activeProject}/analysis/${analysisId}/pillars/${pillarAnalysisToDelete}/`,
+            method: 'DELETE',
+            onSuccess: () => {
+                pillarGetTrigger();
+            },
+            autoTrigger: false,
+        },
+    );
+
+    const handleClick = useCallback(() => {
+        setIsExpanded(!isExpanded);
+    }, [isExpanded]);
+
+    const handlePillarAnalysisToDelete = useCallback((toDeleteKey) => {
+        setPillarAnalysisToDelete(toDeleteKey);
+        deletePillarTrigger();
+    }, [deletePillarTrigger]);
+
+    const analysisPillarRendererParams = useCallback((key, data) => ({
+        pillarId: data.id,
+        title: data.title,
+        assigneeName: data.assigneeName,
+        createdOn: data.createdOn,
+        analysis: data.analysis,
+        setDeleteId: handlePillarAnalysisToDelete,
```

Comment:
type mismatch between the values passed through rendererParams and the AnalysisPillarRendererProps interface. Since the returned values supplied through rendererParams are passed as props to AnalysisPillarRenderer they should have a matching type.